instance/tests/models/test_openedx_appserver.py

```diff
@@ -134,6 +134,7 @@ def test_provision_unhandled_exception(self, mocks):
         self.assertFalse(result)
         mocks.mock_provision_failed_email.assert_called_once_with("AppServer deploy failed: unhandled exception")
 
+    # pylint: disable=too-many-locals
```

Comment:
The threshold for `too-many-locals` [is 15](http://pylint-messages.wikidot.com/messages:r0914). By my count, there are 16 in this function. Getting rid of `instance` and `organization` on lines 178, 179 would fix the linter.

The purpose is to discourage making functions that do too much. This function borderline does too much, so if we can get rid of the `#pylint: disable` we can force the next person who adds to this function to also refactor it.